packages/firestore/test/unit/local/encoded_resource_path.test.ts

```diff
@@ -184,6 +184,7 @@ async function assertOrdered(paths: ResourcePath[]): Promise<void> {
 
   const selected: string[] = [];
   await runTransaction(simpleStore => {
+    selected.splice(0); // The function might get re-run.
```

Comment:
This would be cleaner if the selected array were just returned from the transaction.

Reply:
That's much cleaner. Updated.